tests/test_notifications.py

```diff
@@ -974,20 +978,20 @@ def test_send_email_transactional(self, send_mail):
 
         emails.email_transactional(
             subscribed_users, self.project._id, 'comments',
-            nodeType='project',
+            user=self.project.creator,
+            node=self.project,
             timestamp=timestamp,
-            commenter=self.project.creator,
+            nodeType='project',
```

Comment:
Why is `nodeType` passed? I don't see it used in the transactional or digest email templates? @samanehsan